src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -14591,5 +14575,199 @@ static int Test1(out int x)
 
             Assert.Equal("System.Int32", model.GetTypeInfo(yRef).Type.ToTestDisplayString());
         }
+
+        [Fact, WorkItem(13219, "https://github.com/dotnet/roslyn/issues/13219")]
+        public void IndexingDynamic()
+        {
+            var text = @"
+public class Cls
+{
+    public static void Main()
+    {
+        dynamic d = null;
+        var x = d[out int z];
+    }
+}";
+            // the C# dynamic binder does not support ref or out indexers, so we don't run this
```

Comment:
If the dynamic binder doesn't support ref or out indexers, why doesn't the compiler report an error?